example/android/kotlinlib/1-hello-kotlin/build.mill

```diff
@@ -27,10 +29,41 @@ object app extends AndroidAppKotlinModule {
   def androidSdkModule = mill.define.ModuleRef(androidSdkModule0)
 
   // Configuration for ReleaseKey
+  override def releaseKeyPath = root
   def androidReleaseKeyName: T[String] = Task { "releaseKey.jks" }
   def androidReleaseKeyAlias: T[String] = Task { "releaseKey" }
   def androidReleaseKeyPass: T[String] = Task { "MillBuildTool" }
   def androidReleaseKeyStorePass: T[String] = Task { "MillBuildTool" }
+  override def virtualDeviceIdentifier: String = "kotlin-test"
+  override def emulatorPort: String = "5556"
+
+  /* TODO this won't be needed once the debug keystore is implemented */
+  private def mainRoot = root
+
+  object test extends AndroidAppKotlinTests with TestModule.Junit4 {
+    def ivyDeps = super.ivyDeps() ++ Agg(
+      ivy"junit:junit:4.13.2"
+    )
+  }
+
+  object it extends AndroidAppKotlinIntegrationTests with AndroidTestModule.AndroidJUnit {
+
+    override def instrumentationPackage = "com.helloworld.app"
+
+    /* TODO this needs to change to something better once integration tests work with debug keystore */
+    override def releaseKeyPath = mainRoot
+    def androidReleaseKeyName: T[String] = Task { "releaseKey.jks" }
+    def androidReleaseKeyAlias: T[String] = Task { "releaseKey" }
+    def androidReleaseKeyPass: T[String] = Task { "MillBuildTool" }
+    def androidReleaseKeyStorePass: T[String] = Task { "MillBuildTool" }
```

Comment:
Should we make `AndroidAppKotlinIntegrationTests`  and `AndroidAppIntegrationTests` forward these by default to the enclosing `AndroidAppModule`. In these examples it seems like we always set them to the same value anyway, so might as well make that the default and anyone who wants something different can configure it accordingly

Reply:
bear in mind that this will change completely on the next iteration, as the android tests (in gradle + android studio) generate a debug release key (with android as username and password, perhaps we'll do a mill-android credential pair). It will be part of the work for generating a debug APK which I stubbed here and there in this PR

Reply:
got it, I'm fine leaving it as is then

Reply:
I've changed it anyway before seeing your comment and added more TODO comments around the areas this will need re-work